apps/quick-learn-backend/src/routes/users/users.service.ts

```diff
@@ -121,13 +121,14 @@ export class UsersService extends PaginationService<UserEntity> {
     if (paginationDto.mode == 'paginate') {
       const results = await this.paginate(paginationDto, conditions, [
         ...userRelations,
+        'updated_by',
       ]);
       this.sortByLastLogin(results.items);
       return results;
     }
     const users = await this.userRepository.find({
       where: conditions,
-      relations: [...userRelations],
+      relations: [...userRelations, 'updated_by'],
```

Comment:
This  relation `updated_by` is not needed in every API call, it's will be better if this will be passed as a parameter.